Server/src/main/java/net/simon987/server/GameServer.java

```diff
@@ -76,6 +78,12 @@ public GameServer() {
 
         dayNightCycle = new DayNightCycle();
 
+        SecretKeyGenerator keyGenerator = new SecretKeyGenerator();
+        secretKey = config.getString("secret_key");
+        if (secretKey == null) {
+            secretKey = keyGenerator.generate();
```

Comment:
The generated key should be saved to the config file if it is not present. I added methods to modify the config in 0845438297d742379f5054daf9d5e6dc8f1ce2e2.